### PR TITLE
Branch random event

### LINKED ISSUE #51: Create random events
Create random event at the end of each round of the game to add uncertainty to user's gameplay experience.

### PR DESCRIPTION
**Pull request by ZMinghuiZ**
Add random event to make the game more interesting:). Fixes #51

### CODE REVIEW

File: src/main/java/randomevent/DecisionEvent.java [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+            throw new IllegalArgumentException("Invalid input. Please try again.");
+        }
+    }
+
+    private void presentProject(PlayerProfile playerProfile) throws IllegalArgumentException {
+        System.out.println(DECISIONS[0]);
+        Scanner userInput = new Scanner(System.in);
+        String response = userInput.nextLine();
+        if (isAccept(response)) {
+            switch (new Random().nextInt(2)) {
```
Comment (ZMinghuiZ) [inline src/main/java/randomevent/DecisionEvent.java:79]: Is there any better way for random event generation?

**Review by ZMinghuiZ — COMMENTED** [review]
Any code standard breach?

**Review by StargazerX0 — APPROVED** [review]
well done. By adding more random event, our game can be much more interesting to play

File: src/main/java/randomevent/DecisionEvent.java [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+            case 0:
```
Comment (CXIA17) [inline src/main/java/randomevent/DecisionEvent.java:80]: Interesting feature

**Review by CXIA17 — APPROVED** [review]

**Pull request merged**